src/config.js

```diff
@@ -1069,6 +1119,17 @@ class Config {
     })
   }
 
+  onDidUpdateKeyPath (keyPath, callback) {
+    const oldValue = this.get(keyPath)
```

Comment:
On lines 1128 and 1372 you try to assign `oldValue`. To do this, you need to remove `const`.
Before:
`const oldValue = this.get(keyPath)`
After:
`oldValue = this.get(keyPath)`